gamemodes/attdef.pwn

```diff
@@ -13,6 +13,10 @@
 #include <sampac> 			// THE MIGHTY NEW ANTICHEAT
 #include <mSelection>       // Selection with preview models feature library
 #include <gBugFix>			// Fix false vehicle entry as passenger (G (teleport/distance) bug)
+#include <md-sort>          // Sorts multi dimensional arrays
```

Comment:
Maybe we should use this to sort players results at the end of the match too in the future.